cmd/cmd.go

```diff
@@ -95,106 +97,30 @@ func CreateHandler(cmd *cobra.Command, args []string) error {
 				return err
 			}
 
-			// TODO make this work w/ adapters
 			if fi.IsDir() {
-				tf, err := os.CreateTemp("", "ollama-tf")
-				if err != nil {
-					return err
-				}
-				defer os.RemoveAll(tf.Name())
```

Comment:
I see that this removal of the temp file is no longer done. I know that files in the temp dir will be cleaned up automatically, but could it still be nice to remove these files earlier due to their size?

Reply:
It's still there (line 107) but outside of tempZipFiles since removing it in that function negates all that hard work